data/applications.yml

```diff
@@ -3,250 +3,305 @@
   type: Supporting apps
   team: "#govuk-platform-health"
+  production_hosted_on: carrenza
 
 - github_repo_name: router
   type: Supporting apps
   team: "#govuk-platform-health"
+  production_hosted_on: carrenza
 
 # frontend
 
 - github_repo_name: calculators
   type: Frontend apps
   team: "#govuk-platform-health"
+  production_hosted_on: carrenza
 
 - github_repo_name: calendars
   type: Frontend apps
   team: "#govuk-platform-health"
+  production_hosted_on: carrenza
 
 - github_repo_name: collections
   type: Frontend apps
   team: "#govuk-platform-health"
   component_guide_url: https://govuk-collections.herokuapp.com/component-guide
+  production_hosted_on: carrenza
 
 - github_repo_name: email-alert-frontend
   type: Frontend apps
   team: "#govuk-platform-health"
+  production_hosted_on: carrenza
 
 - github_repo_name: feedback
   type: Frontend apps
   team: "#govuk-platform-health"
+  production_hosted_on: carrenza
 
 - github_repo_name: finder-frontend
   type: Frontend apps
   team: "#govuk-platform-health"
   component_guide_url: https://finder-frontend.herokuapp.com/component-guide
+  production_hosted_on: carrenza
 
 - github_repo_name: frontend
   type: Frontend apps
   team: "#govuk-platform-health"
+  production_hosted_on: carrenza
 
 - github_repo_name: government-frontend
   type: Frontend apps
   team: "#govuk-platform-health"
   component_guide_url: https://government-frontend.herokuapp.com/component-guide
+  production_hosted_on: carrenza
 
 - github_repo_name: info-frontend
   type: Frontend apps
   team: "#govuk-platform-health"
+  production_hosted_on: carrenza
 
 - github_repo_name: licence-finder
   type: Frontend apps
   puppet_name: licencefinder
   team: "#govuk-platform-health"
+  production_hosted_on: carrenza
 
 - github_repo_name: manuals-frontend
   type: Frontend apps
   team: "#govuk-platform-health"
+  production_hosted_on: carrenza
 
 - github_repo_name: smart-answers
   type: Frontend apps
   team: "#govuk-platform-health"
   puppet_name: smartanswers
+  production_hosted_on: carrenza
 
 - github_repo_name: service-manual-frontend
   team: "#govuk-platform-health"
   type: Frontend apps
+  production_hosted_on: carrenza
 
 - github_repo_name: static
   type: Frontend apps
   team: "#govuk-platform-health"
+  production_hosted_on: carrenza
 
 # Transition
 
 - github_repo_name: bouncer
   type: Transition apps
   team: "#govuk-platform-health"
+  production_hosted_on: aws
 
 - github_repo_name: transition
   type: Transition apps
   team: "#govuk-platform-health"
+  production_hosted_on: aws
 
 # data.gov.uk
 
 - github_repo_name: datagovuk_find
   type: data.gov.uk apps
   team: "#govuk-platform-health"
+  production_hosted_on: paas
 
 - github_repo_name: datagovuk_publish
   type: data.gov.uk apps
   team: "#govuk-platform-health"
+  production_hosted_on: paas
 
 - github_repo_name: ckanext-datagovuk
   type: data.gov.uk apps
   team: "#govuk-platform-health"
+  production_hosted_on: paas
```

Comment:
I think this is actually in AWS (@brucebolt would know for sure).

Reply:
Yes, it's on AWS

Reply:
```suggestion
  production_hosted_on: aws
```